src/components/spreadsheet/table-wrapper.tsx

```diff
@@ -485,17 +495,55 @@ const TableWrapper: FunctionComponent<TableWrapperProps> = ({
         });
     }, [sortConfig, mergedColumnData]);
 
+    const toSpreadsheetEquipmentType = useCallback(
+        (tableName: string): SpreadsheetEquipmentType | undefined => {
+            return tablesDefinitions.find((spreadsheetTabDefinition) => spreadsheetTabDefinition.name === tableName)
+                ?.type;
+        },
+        [tablesDefinitions]
+    );
+
     useEffect(() => {
         if (disabled || !equipments) {
             return;
         }
+
+        const equipmentType = toSpreadsheetEquipmentType(tablesNames[tabIndex]);
```

Comment:
This additionnal treatment and use of new selectors won't be necessary if you use spreadsheetNetwork directly

Reply:
Yes, we'll see that in another ticket